spring-cloud-dataflow-docs/src/main/asciidoc/api-guide.adoc

```diff
@@ -2643,3 +2644,118 @@ include::{snippets}/runtime-apps-documentation/list-all-apps/curl-request.adoc[]
 
 include::{snippets}/runtime-apps-documentation/list-all-apps/http-response.adoc[]
 
+
+[[api-guide-resources-stream-logs]]
+=== Stream Logs
+
+It is possible to get the application logs of the stream for the entire stream or a specific application inside the stream.
+The following topics provide more detail:
+
+* <<api-guide-resources-stream-logs-by-stream-name>>
+* <<api-guide-resources-stream-logs-by-app-name>>
+
+
+
+[[api-guide-resources-stream-logs-by-stream-name]]
+==== Get the applications' logs by the stream name
```

Comment:
Get the application logs by stream name

Reply:
I intend to use `applications'` to mean multiple applications logs :-)